op-e2e/e2eutils/disputegame/game_helper.go

```diff
@@ -113,7 +113,7 @@ func (g *FaultGameHelper) WaitForGameStatus(ctx context.Context, expected Status
 	g.t.Logf("Waiting for game %v to have status %v", g.addr, expected)
 	ctx, cancel := context.WithTimeout(ctx, time.Minute)
 	defer cancel()
-	err := utils.WaitFor(ctx, time.Second, func() (bool, error) {
+	err := utils.WaitFor(ctx, 3*time.Second, func() (bool, error) {
```

Comment:
We shouldn't need to check less often which is what the second option is controlling here - it will just make the test slower.  The timeout is set on the context.